sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -224,9 +242,39 @@ static rom_error_t mask_rom_boot(const manifest_t *manifest,
   sec_mmio_check_values(rnd_uint32());
   sec_mmio_check_counters(/*expected_check_count=*/3);
 
+// Configure address translation, compute the epmp regions and the entry
```

Comment:
nit: The spacing of this comment is not aligned with the code block. May be an issue with whitespace.

Reply:
fixed